pkg/sbom/implementation.go

```diff
@@ -155,7 +156,7 @@ func addPackage(doc *spdx.Document, p *pkg) {
 	}
 
 	// Add the purl to the package
-	const extRefCatPackageManager = "PACKAGE_MANAGER"
+	const extRefCatPackageManager = "PACKAGE-MANAGER"
```

Comment:
nit, why not hoist to package level constant for easier discoverability?

Reply:
I definitely can — I thought about this, and wasn't sure of our stylistic preference. A theme from our recent style guide discussions is to keep value definitions as local as possible to where they're used. But this seems like a general SPDX-ism that's perhaps worthy of hoisting

Reply:
7209756